lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,104 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import { faCircle } from '@fortawesome/free-solid-svg-icons';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-
-import NxThreatIndicator from '../NxThreatIndicator';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import NxToolip from '../../NxTooltip/NxTooltip';
+import NxThreatIndicator, { Props } from '../NxThreatIndicator';
+import { rtlRender, userEvent } from '../../../__testutils__/rtlUtils';
+import { screen } from '@testing-library/react';
+import { allThreatLevelCategories } from '../../../util/threatLevels';
 
 describe('NxThreatIndicator', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxThreatIndicator, {});
+  const quickRender = rtlRender<Props>(NxThreatIndicator, {});
 
-  it('renders an faCircle NxFontAwesomeIcon with the `nx-threat-indicator` class', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
-
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toExist();
-    expect(icon).toHaveProp('icon', faCircle);
+  it('renders a threat indicator svg', function() {
+    const { container } = quickRender(),
+        threatIndicator = container.querySelector('svg');
+    expect(threatIndicator).toBeInTheDocument();
   });
 
-  it('sets the nx-threat-indicator--unspecified class if no props are passed', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
+  it('renders a threat indicator if threatLevelCategory prop is provided', async function() {
+    const { container } = quickRender({ threatLevelCategory: 'severe' }),
+        user = userEvent.setup(),
+        threatIndicator = container.querySelector('svg')!;
 
-    expect(icon).toMatchSelector('.nx-threat-indicator--unspecified');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Severe');
   });
 
-  it('sets the modifier class using the threatLevelCategory if it is provided', function() {
-    const icon = getShallowComponent({ threatLevelCategory: 'low' }).find(NxFontAwesomeIcon);
-
-    expect(icon).toMatchSelector('.nx-threat-indicator--low');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--unspecified');
-  });
+  it('renders a threat indicator if policyThreatLevel prop is provided', async function() {
```

Comment:
Unless there's a condition that causes it not to render anything, these two tests don't really make sense to me

Reply:
> Unless there's a condition that causes it not to render anything, these two tests don't really make sense to me

Agreed. Removed here: c5ebedea07e624cc6239aaac7bad0fe8fce4ba18